README.md

```diff
@@ -1,4 +1,69 @@
 # potku-console
-Minimalistic server auto start and configuration platform designed to enable running servers only when needed without affecting user experience drastically.
 
-Very much a WIP project, we are currently working on basic documentation and setting up a proper repository.
+"Minimalistic server auto start and configuration platform."
+
+
+
+## Table of content
+* [Status of the project](#status-of-the-project)
+* [What is potku-console](#what-is-potku-console)
+  * [Pitch for the potku-console](#pitch-for-the-potku-console)
+  * [Main (upcoming) features](#main-(upcoming)-features)
+* [Installation instructions](#installation-instructions)
+* [Technologies and architecture](#technologies-and-architecture)
+* [Getting involved and contributions](#getting-involved-and-contributions)
+
+
+
+## Status of the project
+
+This is currently very much a WIP-project. Core team is currently working on basic architecture and documentation of the project.
+
+
+
+## What is potku-console
+
+Potku-console is a service control platform that is user for starting services automatically on demand. Potku-console consists of different kind of modules like: core, frontend, service modules and cloud modules that together form the potku-console. This repository concentrates especially on core and frontend parts, because they form the spine of the potku-console, but also most important first party service and cloud modules are developed in this repository.
```

Comment:
...that is useD for...
...consists of different kindS of modules...
Maybe remove the like before module listing, IMO would make the sentence flow better.
...especially on THE core and frontend parts...
Maybe remove the also from last sentence and add "as well" to the end.